### PR TITLE
Add routing docs

### PR DESCRIPTION
**Pull request by pfirpfel**
Docs for [ngeo routing feature](https://github.com/camptocamp/ngeo/pull/2669).

### CODE REVIEW

**Comment by pfirpfel** [comment]
Probably messed up the rebase and re-introduced  doc/integrator/features.rst.mako

Since I haven't set up c2cgeoportal locally, I cannot build my docs.

File: doc/integrator/features.rst.mako [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+   api
+   https
+   shortener
+   reset_password
+   wmstime
+   intranet
+   security
+   pdfreport
+   legacy
+   routing
```
Comment (sbrunner) [inline doc/integrator/features.rst.mako]: `features.rst.mako` is renamed to `features.rst`...

File: doc/integrator/routing.rst [hunk 2]
```diff
@@ -80,8 +81,8 @@ profile: string, url-path of the profile, example::
 If more than one profile are supplied, the component will show a drop-down-menu for the user to select a profile.
 
 
-searchDefaultParams
-^^^^^^^^^^^^^^^^^^^
+ngeoNominatimSearchDefaultParams
```
Comment (sbrunner) [inline doc/integrator/routing.rst]: For me the documentation about nominatim should be in ngeo, not in c2cgeoportal...

Comment (pfirpfel) [reply doc/integrator/routing.rst]: It's a setting an integrator would care about when using the routing feature. For example if you don't restrict the country, you will get a lot of useless match from all around the world.

**Comment by pfirpfel** [comment]
@sbrunner Squashed and rebased with master, travis passes now.

**Pull request merged**